### DISCUSSION TITLE
dark mode on windows

### DISCUSSION THREAD

**Discussion by phanirithvij** [espanso/espanso#1671] [Feature requests and Ideas]
For the searchbar dark mode is not working (Neither for the start screen and I have dark mode enabled system wide) on windows 10.
But it works on macos and linux (as smeech pointed out below).

Name        : espanso
Version     : 2.1.8

Also tried latest commit build from [github actions artifacts](https://github.com/espanso/espanso/actions/runs/4929430628)

https://www.reddit.com/r/espanso/comments/vc4lui/dark_mode/
https://www.reddit.com/r/espanso/comments/wgsiv5/how_to_enable_dark_searchbar_as_seen_on_the_home/

**Comment by flappysquirrel** [comment]
Have been looking everywhere. Doesn't look like it's possible.

**Reply by beedell-roke** [reply]
> https://github.com/espanso/espanso/discussions/1671#discussioncomment-7197914

@flappysquirrel, have you read https://learn.microsoft.com/en-us/windows/apps/desktop/modernize/ui/apply-windows-themes#enable-support-for-switching-color-modes? [^1] Irrespective, https://stackoverflow.com/a/63160006/9731176 might provide a method accomplishing this.

[^1]: https://github.com/MicrosoftDocs/windows-dev-docs/blob/dfee5de553268484f06b540b9dc49d75d1397d3f/hub/apps/desktop/modernize/ui/apply-windows-themes.md#enable-support-for-switching-color-modes

**Comment by ghz** [comment]
+1

**Comment by zachpoblete** [comment]
Would love to see this implemented!

**Comment by smeech** [comment]
> But it works on macos.

And Linux.

**Comment by gavindekock** [comment]
+1

**Comment by florinf-ro** [comment]
+1 ... seems like a very old request never to be addressed ...?

**Comment by smeech** [comment]
There are many requests that haven't been implemented, I'm afraid. Federico passed on the project after it lay fallow for a couple of years, and the current development team is very small. 

Any help would be appreciated! 🙂

**Comment by AucaCoyan** [comment]
created #2206
